### PR TITLE
change UI of post page

### PR DESCRIPTION
**Pull request by AilsaF**
- Done modifying UI of post page (including layout and font style)
- Also create CourseCategory2 variable to embrace CourseCode and MultipleCourseCode

Before:
2017-10-15 10 23 28 pm

After:
2017-10-15 10 33 36 pm

### CODE REVIEW

**Comment by AilsaF** [comment]
2017-11-04 6 38 44 pm

File: app/Css/Post.hs [hunk 1]
```diff
@@ -24,16 +24,17 @@ postStyles = do
 tabsCSS :: Css
 tabsCSS = do
     "#posts" & do
-        fontFamily ["HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", "Lucida Grande"][sansSerif]
-        fontSize (px 17)
-        width (pct 97)
+        fontFamily ["Work Sans"][sansSerif]
+        fontSize (px 15)
+        -- width (pct 97)
```
Comment (david-yz-liu) [inline app/Css/Post.hs]: Your changes are all good. You can delete the commented-out CSS now.

File: app/Css/Post.hs [hunk 2]
```diff
@@ -60,16 +61,37 @@ tabsCSS = do
...
+                fontSize (px 15)
             ".credits_not_completed" ? do
-                color red
+                color mRed
+                fontSize (px 15)
 
 postCSS :: Css
 postCSS = do
+    ".year_name" ? do
+        fontFamily ["Work Sans"][sansSerif]
```
Comment (david-yz-liu) [inline app/Css/Post.hs]: Put a space between the two lists (and below as well).

File: app/Css/Post.hs [hunk 3]
```diff
@@ -83,34 +105,46 @@ postCSS = do
         semiVisible
         ":hover" & do
             fullyVisible
+    ".code:before" ? do
+            "content" -: "'✔'"
```
Comment (david-yz-liu) [inline app/Css/Post.hs]: I got an encoding issue here. Safer to use the CSS code: `"content" -: "'\2713'"` (and below)

File: public/js/post/course_components.js [hunk 4]
```diff
@@ -34,14 +34,14 @@ var CourseCode = React.createClass({
             // special case for calculus requirement since it doesn't fit the same pattern
             return (
                 <p className='code'>
-                    (<span id='mat135' className='courseName' onClick={this.openModal}>MAT135H</span> and
-                    <span id='mat136' className='courseName' onClick={this.openModal}>MAT136H</span>) or
+                    <span id='mat135' className='courseName' onClick={this.openModal}>( MAT135H</span> and
```
Comment (david-yz-liu) [inline public/js/post/course_components.js]: This change isn't great (it makes the parenthesis part of the styling). Instead, I recommend removing the padding from `courseName` elements. You might need to add some new padding in other places, and fix some other text, but that's a better change to make.

**Comment by AilsaF** [comment]
2017-11-19 6 12 44 pm

**Pull request merged**